src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -348,6 +648,101 @@ private static double[] ComputeSequenceDeltas(IReadOnlyList<double> a, IReadOnly
             return delta;
         }
 
+        private static ImmutableDictionary<string, TResult>
+            PermutationFeatureImportance<TMetric, TResult>(
```

Comment:
(nit) I would ensure the order of these parameters match the order of `PermutationFeatureImportance.GetImportanceMetricsMatrix`. It will help in readability when people are trying to make sense of the code.

https://github.com/dotnet/machinelearning/blob/04dda55ab0902982b16309c8e151f13a53e9366d/src/Microsoft.ML.Transforms/PermutationFeatureImportance.cs#L21-L31

Reply:
Good point, I hadn't thought of that at all. I'll make that change.